### PR TITLE
Feature/docker sql

### PR DESCRIPTION
**Pull request by BerendWouters**
Adds docker-compose.yml and an ApplicationDbContext

### CODE REVIEW

File: Startup.cs [hunk 1]
```diff
@@ -20,6 +22,14 @@ public Startup(IConfiguration configuration)
       // This method gets called by the runtime. Use this method to add services to the container.
       public void ConfigureServices(IServiceCollection services)
       {
+         var connection = @"Server=db;Database=Singer;User=sa;Password=singer-2019-password;";
```
Comment (vyruz1986) [inline Startup.cs]: Shouldn't this be taken from `appsettings.Development.json`?

**Review by vyruz1986 — CHANGES_REQUESTED** [review]
If I understand correctly, the `docker-compose.yml` will store data in `.dev/` folder in the workspace (typically the repository location). If that is true then we should add the `.div/*` selector to the `.gitignore` file to prevent someone from accidentally checking in that local dev data.

File: Startup.cs (hunk 1 above)
Comment (BerendWouters) [reply Startup.cs]: Agreed, will fix.

Comment (BerendWouters) [reply Startup.cs]: However, there's already a `*.mdf` in the `.gitignore`

**Pull request merged**